conf/battle/misc.conf

```diff
@@ -150,3 +150,6 @@ disp_servervip_msg: no
 
 // Delay to allow user resend new mail (default & minimum is 1000)
 mail_delay: 1000
+
+// Hides items in favorite tab for npc selling.
```

Comment:
// Hides items from the player's favorite tab from being sold to a NPC. (Note 1)